cogsmisc/customization.py

```diff
@@ -184,11 +184,23 @@ def __init__(self, bot):
                 alias = 'Not defined.'
             else:
                 alias = f'{ctx.prefix}alias {alias_name} {alias}'
-            return await ctx.send(f'**{alias_name}**: ```py\n{alias}\n```')
+            out = f'**{alias_name}**: ```py\n{alias}\n```'
+            out = out if len(out) <= 2000 else f'**{alias_name}**:\nCommand output too long to display.\n' \
+                                               f'You can view your personal aliases (and more) on the dashboard.\n' \
+                                               f'https://avrae.io/dashboard/aliases'
```

Comment:
```suggestion
                                               f'<https://avrae.io/dashboard/aliases>'
```
Probably a good idea to add `<>` around the link so discord doesn't embed it. Looks good otherwise!